src/server/src/test/java/io/cassandrareaper/acceptance/ReaperTestJettyRunner.java

```diff
@@ -17,6 +17,7 @@
 
 package io.cassandrareaper.acceptance;
 
+import io.cassandrareaper.ReaperApplicationConfiguration.DatacenterAvailability;
```

Comment:
until we figure out https://github.com/thelastpickle/cassandra-reaper/pull/641 it's problematic to add production classes to imports here.

Reply:
Removed since I can use `getContext()`